samples/assistant/README.md

```diff
@@ -187,7 +187,6 @@ Also note that the storage of chat history is done via table storage. You may co
         {
           "role": "function",
           "content": "The function call succeeded. Let the user know that you completed the action.",
-          "name": "AddTodo"
```

Comment:
Looks like we somehow lost the `name` property. It would be great if we could add this back since it's useful for auditing/debugging.

Reply:
Added back!